lib/chat/db_structures.livemd

```diff
@@ -1,400 +1,415 @@
-# DB structures
+  # DB structures
 
-## User
+  ## User
 
-### User identity
+  ### User identity
 
-User account represented by name and a private key (generated on first login). It is stored on user side along with room keys(room private keys) that user has.
+  User account represented by name and a private key (generated on first login). It is stored on user side along with room keys(room private keys) that user has.
 
-```javascript
-[
-  [name, private_key],
-  [room_key1, room_key2, ...],
-  [] // contacts
-]
-```
+  ```javascript
+  [
+    [name, private_key],
+    [room_key1, room_key2, ...],
+    [] // contacts
+  ]
+  ```
+
+  ### User list
+
```

Comment:
why to indent everything? diff is unreadable

I was using LiveBook Desktop to create it